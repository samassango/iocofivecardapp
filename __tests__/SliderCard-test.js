import 'react-native';
import React from 'react';
import SliderCard from '../src/components/slidercard';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders slider card component correctly', () => {
    const cardTree = renderer.create(<SliderCard item={{ title: 'Get started', text: 'start here', src: require('../src/assets/1.png') }} />);
    expect(cardTree).toMatchSnapshot();
});
