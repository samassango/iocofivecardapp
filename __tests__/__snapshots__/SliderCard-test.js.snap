// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders slider card component correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#322e37",
      "borderBottomColor": "#f5c911",
      "borderBottomWidth": 5,
      "borderRadius": 10,
      "borderWidth": 0,
      "elevation": 20,
      "height": 667,
      "padding": 20,
      "shadowColor": "#f5c911",
      "shadowOffset": Object {
        "height": 0,
        "width": 3,
      },
      "shadowOpacity": 0.5,
      "shadowRadius": 10,
      "width": 280,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "padding": 20,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/1.png",
        }
      }
      style={
        Object {
          "height": 130,
          "width": 130,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "width": 250,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#fff",
          "fontSize": 30,
          "fontWeight": "bold",
        }
      }
    >
      Get started
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "bottom": 60,
        "position": "absolute",
        "width": 250,
      }
    }
  >
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      nativeID="animatedComponent"
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "center",
          "margin": 10,
          "opacity": 1,
          "paddingLeft": 40,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#fff",
            "fontSize": 15,
            "fontWeight": "bold",
          }
        }
      >
        start here
      </Text>
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "#fff",
              "fontSize": 25,
            },
            undefined,
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        î—Œ
      </Text>
    </View>
  </View>
</View>
`;
